unison-cli/src/Unison/Auth/HTTPClient.hs

```diff
@@ -17,11 +15,11 @@ newtype AuthenticatedHttpClient = AuthenticatedHttpClient HTTP.Manager
 
 -- | Returns a new http manager which applies the appropriate Authorization header to
 -- any hosts our UCM is authenticated with.
-newAuthenticatedHTTPClient :: MonadIO m => (Output v -> IO ()) -> TokenProvider -> UCMVersion -> m AuthenticatedHttpClient
-newAuthenticatedHTTPClient responder tokenProvider ucmVersion = liftIO $ do
+newAuthenticatedHTTPClient :: MonadIO m => TokenProvider -> UCMVersion -> m AuthenticatedHttpClient
```

Comment:
Why don't we want to print these `CredentialFailureMsg`s anymore?

Reply:
Since this is location is triggered pre-request, we print a ton of "you need to re-auth" messages when pulling from public endpoints when really the user doesn't need to be authed for those at all.

There's a check in the response which triggers a "You need to re-auth" if the result is a 401 which is more accurate and covers the same situation 👍🏼